cpp/libcudf_kafka/cmake/thirdparty/CUDF_KAFKA_GetGTest.cmake

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
We should be able to delete this file now that cudf [makes its imported `GTest::` targets global](https://github.com/rapidsai/cudf/pull/7574/files#diff-674de455ad35a39f00ef3568d57bd5141af75b124ce3648bac86b7e1e123164eR36-R40).

Reply:
roger, will delete now